js/main.js

```diff
@@ -375,7 +375,7 @@ jQuery(document).on('ready', function() {
       var talk_id = schedule[i].talk_id;
       var entity_details = schedule[i];
       var title = entity_details.title;
```

Comment:
change this to `var display_title = entity_details.title;` and delete line 381 since that line adds speaker name with the title which we have separately as `speaker_name`

Reply:
This issue has been resolved in d96d50910fbbd21ac5e189461eb1fba7ebf57b2b